### PR TITLE
JSDK-1974 Updating twilio-video.js to 2.x.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@markandrus

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -173,36 +174,48 @@ function participantConnected(participant) {
...
   participantDiv.appendChild(videoElement);
-
   participants.appendChild(participantDiv);
 
-  participant.tracks.forEach(track => trackAdded(participant, track));
-  participant.on('trackAdded', track => trackAdded(participant, track));
-  participant.on('trackRemoved', track => trackRemoved(participant, track));
-  participant.once('disconnected', () => participantDisconnected(participant));
+  participant.tracks.forEach(publication => trackPublished(participant, publication));
+  participant.on('trackPublished', publication => trackPublished(participant, publication));
```
Comment (markandrus) [inline src/index.js:180]: This is instructional, although just setting listeners for "trackSubscribed" and "trackUnsubscribed" would be sufficient, too. I'm happy with this. Would be nice to handle the "trackUnpublished" event, too, I think.

**Review by markandrus — COMMENTED** [review]
I had one minor comment; otherwise, LGTM.

File: src/index.js (hunk 1 above)
Comment (manjeshbhargav) [reply src/index.js:180]: @markandrus Since we don't do anything with "trackUnpublished" here, do you want to just log the event?

Comment (markandrus) [reply src/index.js:180]: Yeah, just logging is fine 👍

**Pull request merged**